events/static/js/signup/components/SignupForm.js

```diff
@@ -106,6 +106,10 @@ class SignupForm extends Component {
 
     return (
         <Grid container spacing={16}>
+          {/* TODO Image have no restriction */}
+        <div className='image-section' >
+          <img alt='' src={event.image_url}/>
+        </div>
```

Comment:
Need help with how to restrict the size of image so it does not overflow the page.

Reply:
@rrudling Can you look into this problem with using #673 Docker instructions? This would be reduce the time needed when I look into why Vagrant is creating the issue